student.rb

```diff
@@ -1,16 +1,20 @@
-class Student
+require_relative "education"
+
+class Student < Education
```

Comment:
A student is an education?  Find a name so that you can say both "A student is a(n) X" and "A teacher is a(n) X"